src/RouteJs.AspNet/router.js

```diff
@@ -253,6 +256,22 @@
 			}
 
 			return parsedConstraints;
+		},
+
+		_shouldConvertParam: function (param) {
+			///<summary>Gets if we should convert this param.</summary>
+			///<param name="param">The param to check</param>
+
+			return param == 'controller'
+			|| param == 'action'
```

Comment:
Nit: Please format this like:
```
return (
  param === 'controller' ||
  param === 'action' ||
  param === 'area'
);
```

(`param` at the same indentation on every line, and `||` at the end of the line).